MapleServer2/Types/StatDistribution.cs

```diff
@@ -3,17 +3,45 @@
 
 namespace MapleServer2.Types;
 
+public struct ExtraSkillPoints
+{
+    public Dictionary<short, int> ExtraPoints;
+
+    public ExtraSkillPoints()
+    {
+        ExtraPoints = new Dictionary<short, int>();
+    }
+}
+
 public class StatDistribution
 {
     public int TotalStatPoints;
     public Dictionary<OtherStatsIndex, int> OtherStats { get; } // Dictionary of OtherStatsIndex and amount of points allocated to it
     public Dictionary<StatAttribute, int> AllocatedStats { get; } // Dictionary of StatId and amount of stat points allocated to it
 
+    public const int MaxSkillSources = 3;
+    public const short MaxSkillJobRanks = 2;
+    public int TotalSkillPoints;
+    public Dictionary<OtherStatsIndex, ExtraSkillPoints> ExtraSkillPoints { get; }
```

Comment:
I would prefer if you made a new enum since OthersStatsIndex has more than what skills will use.

1 Trophy
2 Chapter
3 Unk